src/clj/game/cards/hardware.clj

```diff
@@ -398,13 +398,14 @@
     (let [ability {:msg (msg "move 1 virus counter to " (:title target))
                    :req (req (pos? (get-in card [:counter :virus] 0)))
                    :choices {:req #(and (has-subtype? % "Virus")
-                                        (pos? (get-in % [:counter :virus] 0)))}
-                   :effect (req (when (pos? (get-virus-counters state side target))
-                                  (add-counter state side card :virus -1)
-                                  (add-counter state side target :virus 1)))}]
+                                        (is-type? % "Program"))}
+                   :effect (req (add-counter state :runner card :virus -1)
+                                (add-counter state :runner target :virus 1))}]
      {:events {:runner-turn-begins ability
-               :runner-trash {:msg "to gain a virus counter"
-                              :effect (effect (add-counter card :virus 1))}}})
+               :runner-trash {:optional
+                              {:prompt "Gain a virus counter on Friday Chip?"
+                               :yes-ability {:msg "to gain a virus counter"
```

Comment:
Keep in mind this `:msg` will result in a message like "Player uses Friday Chip to to gain a virus counter"

Reply:
Hmm. Maybe I should do it with a `system-msg` as `places 1 virus counter on Friday Chip`.

Reply:
My point was more about the doubled "to", but I've used `system-msg` like that often to indicate results of automatic behavior rather than active usage.